app/src/main/res/layout/search_category.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2019 Derek Ellis. Subject to the MIT license.
+  -->
+
+<androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <TextView
+            android:id="@+id/search_category"
```

Comment:
We should pick one id naming scheme and stick to it. With this PR we'll have both `camelCase` and `snake_case` ids.

Reply:
idk i like snake_case since xml file names are snake_case anyway
Otherwise we'd have 
``` Kotlin
setContentView(R.layout.activity_search)
findViewById(R.id.someCamelCaseId)
```

Reply:
Good point. We'll stick with `snake_case` for layout id's then. I'll get around to updating the other layouts.